cognite/client/data_classes/transformations/__init__.py

```diff
@@ -68,6 +72,73 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
             return convert_all_keys_to_camel_case(ret)
         return ret
 
+class SessionSourceDetails:
+    """Details of a session which provid.
+
+    Args:
+        session_id (int): CDF session ID
+        client_id (str): Idp client ID
+        project_name (str): CDF project name
+    """
+
+    def __init__(
+        self,
+        session_id: int = None,
+        client_id: str = None,
+        project_name: str = None,
+    ):
+        self.session_id = session_id
+        self.client_id = client_id
+        self.project_name = project_name
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        """Dump the instance into a json serializable Python data type.
+
+        Args:
+            camel_case (bool): Use camelCase for attribute names. Defaults to False.
+
+        Returns:
+            Dict[str, Any]: A dictionary representation of the instance.
+        """
+        ret = vars(self)
+
+        if camel_case:
+            return convert_all_keys_to_camel_case(ret)
+        return ret
+
+class SessionDestinationDetails:
+    """Details of a session which provid.
+
+    Args:
+        session_id (int): CDF session ID
+        client_id (str): Idp client ID
+        project_name (str): CDF project name
+    """
+
+    def __init__(
+        self,
+        session_id: int = None,
+        client_id: str = None,
+        project_name: str = None,
+    ):
+        self.session_id = session_id
+        self.client_id = client_id
+        self.project_name = project_name
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        """Dump the instance into a json serializable Python data type.
+
+        Args:
+            camel_case (bool): Use camelCase for attribute names. Defaults to False.
+
+        Returns:
+            Dict[str, Any]: A dictionary representation of the instance.
+        """
+        ret = vars(self)
+
+        if camel_case:
+            return convert_all_keys_to_camel_case(ret)
+        return ret
```

Comment:
These classes are exactly the same as `SessionDetails`, so they're redundant. Please remove them and use `SessionDetails` instead.